src/book/03-guides/04-general/04-documentation.mdx

```diff
@@ -0,0 +1,86 @@
+---
+section: Guides
+chapter: General
+title: Documentation
```

Comment:
```suggestion
title: Code Documentation
```
Since this only covers code documentation and I would assume a documentation page to cover more than that.